src/model.rs

```diff
@@ -159,6 +164,9 @@ pub enum StakeCredential {
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
 pub struct BlockRecord {
+    #[serde(serialize_with = "serialize_era")]
+    #[serde(deserialize_with = "deserialize_era")]
+    pub era: Era,
```

Comment:
@scarmuega just FYI this won't compile until https://github.com/txpipe/pallas/pull/53 lands